src/pages/addFood/index.jsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+import { Button, Form, Input, Select, Space, Upload } from 'antd';
+import { UploadOutlined } from '@ant-design/icons';
+import TextArea from 'antd/es/input/TextArea';
+
+import './index.css';
+import { useAddFoodData } from '../../hooks/useFoodsData';
+import BackButton from '../../components/backButton';
+
+const AddFood = () => {
+  const [name, setName] = useState('');
+  const [category, setCategory] = useState('');
+  const [price, setPrice] = useState(0);
+  const [description, setDescription] = useState('');
+  const [rating, setRating] = useState(0);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [image, setImage] = useState([]);
+
+  const { mutate: addFood } = useAddFoodData();
+
+  const normFile = (e) => {
+    if (Array.isArray(e)) {
+      setImage(e);
+      return e;
+    }
+    setImage(e?.fileList);
+    return e?.fileList;
+  };
+
+  const handleAddFood = () => {
+    if (
+      name.trim() === '' ||
```

Comment:
why using trim here?

Reply:
if any user adds just spaces or extra spaces in input box then it will remove it.